src/SFA.DAS.Employer.PR.Web/Controllers/YourTrainingProvidersController.cs

```diff
@@ -5,19 +5,30 @@
 using SFA.DAS.Employer.PR.Web.Authentication;
 using SFA.DAS.Employer.PR.Web.Extensions;
 using SFA.DAS.Employer.PR.Web.Infrastructure;
+using SFA.DAS.Employer.PR.Web.Infrastructure.Services;
 using SFA.DAS.Employer.PR.Web.Models;
+using SFA.DAS.Employer.PR.Web.Models.Session;
 
 namespace SFA.DAS.Employer.PR.Web.Controllers;
 
 [Authorize(Policy = nameof(PolicyNames.HasEmployerAccount))]
 [Route("accounts/{employerAccountId}/providers", Name = RouteNames.YourTrainingProviders)]
-public class YourTrainingProvidersController(IOuterApiClient _outerApiClient) : Controller
+public class YourTrainingProvidersController(IOuterApiClient _outerApiClient, ISessionService _sessionService) : Controller
 {
     public async Task<IActionResult> Index([FromRoute] string employerAccountId, CancellationToken cancellationToken)
```

Comment:
HttpGet missing, also should we stick to one naming convention, if you want to use Index, then that is fine, but elsewhere you have used Get and Post, you can use Index for both Get and Post as long as you mark them with correct attribute.

Reply:
I think I'll go with Index and use verbs to differentiate